back/engines/commercial/machine_translations/app/controllers/machine_translations/web_api/v1/machine_translations_controller.rb

```diff
@@ -44,6 +44,12 @@ def show
 
               render json: { errors: { base: [{ error: 'translatable_blank' }] } }, status: :unprocessable_entity
               return
+            rescue EasyTranslate::EasyTranslateException => e
+              raise e unless e.message.include?('Bad language pair')
+
+              render json: { errors: { base: [{ error: 'bad_language_pair' }] } }, status: :unprocessable_entity
+              skip_authorization
```

Comment:
This prevents a `Pundit::AuthorizationNotPerformedError`, but something feels off about doing this here, and in a different case above, when other rescue case don't seem to need it. Any ideas for a more elegant solution?

Reply:
No, I mean the `skip authorization`. I am doing this in a second place now, and this seems wrong to me, but I'm not sure how to refactor to avoid this.